cardano-cli/src/Cardano/CLI/Shelley/Output.hs

```diff
@@ -0,0 +1,47 @@
+module Cardano.CLI.Shelley.Output
```

Comment:
@newhoggy This would have been better placed in `Cardano.CLI.Types` rather than creating a separate module.